src/main/java/factionsplusplus/commands/MembersCommand.java

```diff
@@ -59,7 +59,7 @@ public void execute(CommandContext context) {
             this.constructMessage("MembersFaction.Title")
                 .with("faction", faction.getName())
         );
-        faction.getMembers().stream()
+        faction.getMembersUUIDS().stream()
```

Comment:
Keep this as getMembers() then call .map(GroupMember::getUUID) immediately after on the stream to convert them all to UUIDs